WordPress/Classes/ViewRelated/NUX/SigninEditingState.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
 class SigninEditingState {
-    static var signinEditingStateActive = false
+    static var signinEditingStateActive = true
```

Comment:
I can remove this. Just to be clear, this only affects the first text field seen: the email text field. After that, behaviour is exactly as it was previously as `signinEditingStateActive` is set to `false` or `true` as the user moves through the flow.

The implication for Safari saved passwords is that users aren't prompted for them until they tap on the text field. It's a little bit jarring.